src/FSharp.Markdown/MarkdownParser.fs

```diff
@@ -449,6 +449,35 @@ let rec (|ListItems|_|) prevSimple = function
 
 // Code for parsing pipe tables
 
+// Splits table row into deliminated parts escaping inline code and latex
+let rec PipeTableFindSplits (delim : char array) (line : char list) = 
```

Comment:
Matthias,

I made a "minimal change" patch so this is a subfunction used by the 
existing row parser - it is not a recogniser.

It could be inserted as a subfunction of active recogniser PipeTableRow, 
but big subfunctions inside smaller functions are not so readable?

Best wishes, Tom

On 03/04/2016 13:11, Matthias Dittrich wrote:
>
> In src/FSharp.Markdown/MarkdownParser.fs 
> <https://github.com/tpetricek/FSharp.Formatting/pull/394#discussion_r58309211>:
>
> > @@ -449,6 +449,35 @@ let rec (|ListItems|_|) prevSimple = function
> >
> >  // Code for parsing pipe tables
> >
> > +// Splits table row into deliminated parts escaping inline code and latex
> > +let rec PipeTableFindSplits (delim : char array) (line : char list) =
>
> Can we lower case this (or change to an active pattern)?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/tpetricek/FSharp.Formatting/pull/394/files/1fbfa6f43f20dd312d7459c6708ad2b1a12047b9#r58309211>
>